### PR TITLE
Docs: Slightly improve wording of `DROP TABLE` docs

### PR DESCRIPTION
**Pull request by rschu1ze**
Follow-up to #58705

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

---
### Modify your CI run:
**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Unit tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with Analyzer
- [ ]  Add your option here

#### Exclude tests:
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with MSAN
- [ ]  All with UBSAN
- [ ]  All with Coverage
- [ ]  All with Aarch64
- [ ]  Add your option here

#### Extra options:
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ]  1
- [ ]  2
- [ ]  3
- [ ]  4

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit 2f45d98c970740c2263812fe3044616787f49d96 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/62228/2f45d98c970740c2263812fe3044616787f49d96/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Comment by rschu1ze** [comment]
I must be doing something wrong in the PR description, `PullRequestCI / RunConfig (pull_request)` says:

```
[...]
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/test_ci_options.py", line 78, in test_options_applied
    self.assertCountEqual(
AssertionError: Element counts were not equal:
First has 0, Second has 1:  'package_release'
First has 0, Second has 1:  'package_asan'
First has 0, Second has 1:  'package_ubsan'
First has 0, Second has 1:  'package_debug'
First has 0, Second has 1:  'package_msan'
First has 0, Second has 1:  'Stateless tests (asan)'
First has 0, Second has 1:  'Stateless tests flaky check (asan)'
First has 0, Second has 1:  'Stateless tests (msan)'
First has 0, Second has 1:  'Stateless tests (ubsan)'
First has 0, Second has 1:  'Stateless tests (debug)'
First has 0, Second has 1:  'Stateless tests (release)'
First has 0, Second has 1:  'Integration tests (release)'
First has 0, Second has 1:  'Integration tests (asan)'
First has 0, Second has 1:  'Integration tests flaky check (asan)'
======================================================================
FAIL: test_options_applied_2 (test_ci_options.TestCIOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/test_ci_options.py", line 112, in test_options_applied_2
    self.assertCountEqual(
AssertionError: Element counts were not equal:
First has 0, Second has 1:  'Integration tests (asan, old analyzer)'
First has 0, Second has 1:  'package_release'
First has 0, Second has 1:  'Stateless tests (release, old analyzer, s3, DatabaseReplicated)'
First has 0, Second has 1:  'package_asan'
======================================================================
FAIL: test_pr_body_parsing (test_ci_options.TestCIOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/test_ci_options.py", line 53, in test_pr_body_parsing
    self.assertFalse(ci_options.do_not_test)
AssertionError: True is not false
```

File: docs/en/sql-reference/statements/drop.md [hunk 1]
```diff
@@ -20,11 +20,10 @@ DROP DATABASE [IF EXISTS] db [ON CLUSTER cluster] [SYNC]
 
 ## DROP TABLE
 
-Deletes the table.
-In case when `IF EMPTY` clause is specified server will check if table is empty only on replica that received initial query.  
+Deletes one or more tables.
 
 :::tip
-Also see [UNDROP TABLE](/docs/en/sql-reference/statements/undrop.md)
+To undo the deletion of a table, please see see [UNDROP TABLE](/docs/en/sql-reference/statements/undrop.md)
```
Comment (hanfei1991) [inline docs/en/sql-reference/statements/drop.md:26]:
```suggestion
To undo the deletion of a table, please see [UNDROP TABLE](/docs/en/sql-reference/statements/undrop.md)
```

**Review by hanfei1991 — APPROVED** [review]

**Comment by rschu1ze** [comment]
```
2024-04-03T20:29:04.4871941Z ======================================================================
2024-04-03T20:29:04.4922744Z FAIL: test_options_applied (test_ci_options.TestCIOptions)
2024-04-03T20:29:04.4923950Z ----------------------------------------------------------------------
2024-04-03T20:29:04.4924741Z Traceback (most recent call last):
2024-04-03T20:29:04.4926303Z   File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/test_ci_options.py", line 78, in test_options_applied
2024-04-03T20:29:04.4927803Z     self.assertCountEqual(
2024-04-03T20:29:04.4928461Z AssertionError: Element counts were not equal:
2024-04-03T20:29:04.4929331Z First has 0, Second has 1:  'package_release'
2024-04-03T20:29:04.4930161Z First has 0, Second has 1:  'package_asan'
2024-04-03T20:29:04.4930979Z First has 0, Second has 1:  'package_ubsan'
2024-04-03T20:29:04.4931797Z First has 0, Second has 1:  'package_debug'
2024-04-03T20:29:04.4932605Z First has 0, Second has 1:  'package_msan'
2024-04-03T20:29:04.4933495Z First has 0, Second has 1:  'Stateless tests (asan)'
2024-04-03T20:29:04.4934579Z First has 0, Second has 1:  'Stateless tests flaky check (asan)'
2024-04-03T20:29:04.4935646Z First has 0, Second has 1:  'Stateless tests (msan)'
2024-04-03T20:29:04.4936836Z First has 0, Second has 1:  'Stateless tests (ubsan)'
2024-04-03T20:29:04.4937817Z First has 0, Second has 1:  'Stateless tests (debug)'
2024-04-03T20:29:04.4938801Z First has 0, Second has 1:  'Stateless tests (release)'
2024-04-03T20:29:04.4939828Z First has 0, Second has 1:  'Integration tests (release)'
2024-04-03T20:29:04.4940842Z First has 0, Second has 1:  'Integration tests (asan)'
2024-04-03T20:29:04.4941928Z First has 0, Second has 1:  'Integration tests flaky check (asan)'
2024-04-03T20:29:04.4942706Z 
2024-04-03T20:29:04.4942962Z ======================================================================
2024-04-03T20:29:04.4944603Z FAIL: test_options_applied_2 (test_ci_options.TestCIOptions)
2024-04-03T20:29:04.4945674Z ----------------------------------------------------------------------
2024-04-03T20:29:04.4946456Z Traceback (most recent call last):
2024-04-03T20:29:04.4948617Z   File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/test_ci_options.py", line 112, in test_options_applied_2
2024-04-03T20:29:04.4950140Z     self.assertCountEqual(
2024-04-03T20:29:04.4950805Z AssertionError: Element counts were not equal:
2024-04-03T20:29:04.4951876Z First has 0, Second has 1:  'Integration tests (asan, old analyzer)'
2024-04-03T20:29:04.4952929Z First has 0, Second has 1:  'package_release'
2024-04-03T20:29:04.4954187Z First has 0, Second has 1:  'Stateless tests (release, old analyzer, s3, DatabaseReplicated)'
2024-04-03T20:29:04.4955422Z First has 0, Second has 1:  'package_asan'
2024-04-03T20:29:04.4955930Z 
2024-04-03T20:29:04.4956198Z ======================================================================
2024-04-03T20:29:04.4957091Z FAIL: test_pr_body_parsing (test_ci_options.TestCIOptions)
2024-04-03T20:29:04.4958094Z ----------------------------------------------------------------------
2024-04-03T20:29:04.4958893Z Traceback (most recent call last):
2024-04-03T20:29:04.4960616Z   File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/test_ci_options.py", line 53, in test_pr_body_parsing
2024-04-03T20:29:04.4962124Z     self.assertFalse(ci_options.do_not_test)
2024-04-03T20:29:04.4962861Z AssertionError: True is not false
```

**Pull request merged**